src/app/services/notification.service.ts

```diff
@@ -0,0 +1,82 @@
+﻿import { Injectable } from "@angular/core";
+
+import { ToastrService } from "ngx-toastr";
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NotificationService {
+
+  private static readonly _DEFAULT_TIMEOUT: number = 5000;
+  private static readonly _DEFAULT_EXTENDED_TIMEOUT: number = 5000;
+
+
+  constructor(
+    private _toastrService: ToastrService
+  ) {}
+
+
+  /**
+   * Displays the given error message as a toast
+   */
+  displayErrorMessage(message: string, title?: string, timeout?: number): void {
+
+    this._toastrService.error(
+      `${message}`,
+      title || "Error",
+      {
+        timeOut: timeout || NotificationService._DEFAULT_TIMEOUT,
+        extendedTimeOut: timeout || NotificationService._DEFAULT_EXTENDED_TIMEOUT
+      }
+    );
+  }
```

Comment:
Maybe we can make the content cleaner by setting default parameters? 
```suggestion
  displayErrorMessage(message: string, title: string = "Error", timeout?: number): void {

    this._toastrService.error(
      `${message}`,
      title,
      {
        timeOut: timeout || NotificationService._DEFAULT_TIMEOUT,
        extendedTimeOut: timeout || NotificationService._DEFAULT_EXTENDED_TIMEOUT
      }
    );
  }
```